pkg/plugins/platform/local.go

```diff
@@ -174,17 +161,17 @@ func (l *LocalApp) ImportDB(imPath string, extPath string) error {
 		if extPath == "" {
 			extPathPrompt = true
 		}
-		fmt.Println("Provide the path to the database you wish to import.")
-		fmt.Println("Import path: ")
+		util.UserOut.Println("Provide the path to the database you wish to import.")
+		fmt.Print("Import path: ")
```

Comment:
This should be `util.UserOut`, right?

Reply:
Unfortunately the logrus system *always* seems to emit a CR, so I used fmt.Print() on the prompts for config and import commands. Open to other solutions.